Packs/PrismaSaasSecurity/Integrations/SaasSecurity/SaasSecurity.yml

```diff
@@ -50,12 +50,12 @@ configuration:
   type: 15
   required: false
   section: Collect
-- display: Number of incidents per fetch
+- display: Number of incidents per fetch.
   name: max_fetch
   defaultvalue: "20"
   type: 0
   required: true
-  additionalinfo: Minimum is 10. Maximum is 1000.
+  additionalinfo: "Important: The limit value can range from 10 to 200 and must be in multiples of 10."
```

Comment:
```suggestion
  additionalinfo: "Important: The limit value can range from 10 to 200 and must be in multiples of 10. Default is 50"
```

Reply:
This is the default in the fetch-limit parameter in the configuration window so it appears clearly to the user.